test/integration/sequelize/query.test.js

```diff
@@ -342,6 +342,20 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
           await this.UserVisit.sync({ force: true });
         });
 
+        it('emits raw errors if requested', async function() {
+          const sql = 'SELECT 1 FROM NotFoundTable';
+          try {
+            await this.sequelize.query(sql, { rawErrors: false });
+          } catch (err) {
+            expect(err).to.be.instanceOf(DatabaseError);
```

Comment:
Can you check that it is NOT the actual error class?

Reply:
> the actual error class?
Same problem as before, the "actual error class" is dialect specific, and always != sequelize's own DatabaseError.